docs/docs/running-hello-world.md

```diff
@@ -73,265 +85,289 @@ The web server loads the enclave and waits for requests from the client. Amongst
+    ```bash
+    java -jar client/build/libs/client.jar "S:4924CA3A9C8241A3C0AA1A24A407AA86401D2B79FA9FF84932DA798A942166D4 PROD:1 SEC:INSECURE" reverse-me
+    ```
 === "Windows"
+    
+    1. When you run the host, you should see the new signing key in the output:
+    ```bash hl_lines="3"
+    [main] INFO com.r3.conclave.host.web.EnclaveWebController -
+    Remote attestation for enclave A92F481B7EEAE42D3EBB162BF77613605AF214D77D2E63D75A610FD485CFD7D6:
+    - Mode: SIMULATION
+    - Code signer: 4924CA3A9C8241A3C0AA1A24A407AA86401D2B79FA9FF84932DA798A942166D4
+    - Session signing key: 302A300506032B6570032100D23DD5C05A37CB5B6ED50EA1501E55ABF0EF85B50A97A69D0C3F4F84372AF928
+    - Session encryption key: 42CF5E2457B19A9E4FA3716F40CDF6B07A3EEC95D1AFE29C6F1DE99FD0DC647C
+    - Product ID: 1
+    - Revocation level: 0
+    ``` 
+    2. If you haven't built the client already, build it the same way as in mock mode:
     ```bash
     gradlew.bat :client:shadowJar
     ```
-=== "macOS / Linux"
+    3. Run the client using the signing key from the host's attestation report
     ```bash
-    ./gradlew :client:shadowJar
+    java -jar client/build/libs/client.jar "S:4924CA3A9C8241A3C0AA1A24A407AA86401D2B79FA9FF84932DA798A942166D4 PROD:1 SEC:INSECURE" reverse-me
     ```
 
-Then run the client using the signing key from the host's attestation report
-```bash
-java -jar client/build/libs/client.jar \
-  "S:4924CA3A9C8241A3C0AA1A24A407AA86401D2B79FA9FF84932DA798A942166D4 PROD:1 SEC:INSECURE" \
-  reverse-me
-```
-
 ## Next steps
-Now you know how to build and run the hello world sample, see how it is implemented in [Writing hello world](writing-hello-world.md).
+Now you know how to build and run the hello world sample, see how it is implemented in
+[Writing hello world](writing-hello-world.md).
+
+## Got this far? Join the community
+
+You can ask your questions on our Discord server. Conclave's development team answers your questions during UK
+business hours (GMT 0900-1700).
 
-## Got this far? Join the community!
+You can discuss everything about Conclave on our mailing list. You can also use the mailing list to discuss SGX and
+other Trusted Execution Environment (TEE) technologies.
 
-A discord server is available where you can find the development team during UK office hours (GMT 0900-1700).
-There's a public mailing list for users to discuss Conclave, and we also welcome general SGX talk.
 
-[:fontawesome-brands-discord: Join our discord server](https://discord.com/invite/dDaBGqGPre){: .md-button }
+[:fontawesome-brands-discord: Join our discord server](https://discord.gg/zpHKkMZ8Sw){: .md-button }
 
 [:fontawesome-solid-paper-plane: Join conclave-discuss@groups.io](https://groups.io/g/conclave-discuss){: .md-button }
 
-[:fontawesome-solid-paper-plane: Email us directly](mailto:conclave@r3.com){: .md-button }
+[:fontawesome-solid-paper-plane: Email us](mailto:conclave@r3.com){: .md-button }
 
 ## Appendix: Summary of Docker command options
```

Comment:
We should not be explaining docker command options! These are already well documented on Docker's own docsite. Let's please remove this.